app/views/ncr/proposals/new.html.erb

```diff
@@ -2,7 +2,7 @@
   <h4>National Capital Region</h4>
   <h3>Purchase Card Approval Request</h3>
 
-  <%= form_for @proposal_form do |f| %>
+  <%= form_for @proposal_form, url: @form_url, method: @form_method do |f| %>
```

Comment:
I've played with this a bit more and had no luck. I'm going to leave it for future refactoring.